waffle/models.py

```diff
@@ -233,6 +233,9 @@ def is_active_for_user(self, user: AbstractBaseUser) -> bool | None:
         if self.everyone:
             return True
 
+        if self.everyone is False:
```

Comment:
This and the previous if, could be changed to?
```suggestion
        if self.everyone != None:
            return self.everyone
```

Reply:
this does not follow PEP standards